edxval/tests/test_api.py

```diff
@@ -1158,6 +1183,40 @@ def test_basic(self, course_id, image):
         self.assert_xml_equal(exported_metadata['xml'], expected)
         self.assertEqual(sorted(exported_metadata['transcripts'].keys()), ['de', 'en'])
 
+    @data(
+        {'course_id': None, 'image': ''},
+        {'course_id': 'test-course', 'image': 'image.jpg'},
+    )
+    @unpack
+    def test_basic_wrapfs(self, course_id, image):
+        """
+        Test that video export works as expected, with wrapfs as sometimes used by edx-platform.
```

Comment:
nit: move `,` after wrapfs and mention olx_rest_api after edx-platform to be more specific